.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea/
```

Comment:
This should be put in a local (non-committed) gitignore, I think.

Reply:
And I would argue: at the moment, Rider is one of the major development tools I personally use for the project. If we put Visual Studio stuff there, we should put Rider stuff as well. That'll be just convenient for the people working with the project: they won't need to set up anything to ignore any stuff their IDEs are creating.

Before anybody ask: I'd be happy to add MonoDevelop (or Xamarin Studio, or something else) stuff in our .gitignore, too, if somebody uses that IDE.